Globalization/ICU file updates (#36178)
dotnet/AspNetCore.Docs: aspnetcore/blazor/globalization-localization.md

For current browser support of the preceding types, see [Can I use](https://caniuse.com).

> By default, Blazor only loads globalization data for the app's culture. To load all globalization data, set `<BlazorWebAssemblyLoadAllGlobalizationData>` to `true` in the app's project file (`.csproj`):

Author: ```suggestion
By default, Blazor loads a subset of globalization data that contains the app's culture. To load all globalization data, set `<BlazorWebAssemblyLoadAllGlobalizationData>` to `true` in the app's project file (`.csproj`):
```

> By default, Blazor only loads globalization data for the app's culture. To load all globalization data, set `<BlazorWebAssemblyLoadAllGlobalizationData>` to `true` in the app's project file (`.csproj`):

Author: "Shard" isn't defined, so I like your suggestion.
